scripts/test/compactionPerformance/system-test-instance.properties

```diff
@@ -168,59 +90,14 @@ sleeper.compaction.files.batch.size=11
 # Valid values are 'arraylist' and 'arrow'
 sleeper.ingest.record.batch.type=arrow
 
-# The way in which partition files are written to the main Sleeper store
-# Valid values are 'direct' (which writes using the s3a Hadoop file system) and 'async' (which writes locally and then
-# copies the completed Parquet file asynchronously into S3)
-sleeper.ingest.partition.file.writer.type=async
-
-# The maximum number of records that the ingest process writes to the local file (arraylist only)
-sleeper.ingest.max.local.records=100000000
-
-# The maximum number of records that the ingest process loads into memory at one time (arraylist only)
-sleeper.ingest.memory.max.batch.size=1000000
-
-# The number of bytes to allocate to the Arrow working buffer, which is used for sorting and other sundry activities (arrow only) [256MB]
-sleeper.ingest.arrow.working.buffer.bytes=268435456
-
-# The number of bytes to allocate to the Arrow batch buffer, which is used to hold the records (arrow only) [1GB]
-sleeper.ingest.arrow.batch.buffer.bytes=1073741824
-
-# The maximum number of bytes to store on the local disk before uploading to the main Sleeper store (arrow only) [2GB]
-sleeper.ingest.arrow.max.local.store.bytes=2147483648
-
 # The number of records to write at once into an Arrow file in the local store. A single Arrow file will be written to
 # multiple times and so this parameter does not affect the final size of the Arrow file (arrow only) [1K]
 sleeper.ingest.arrow.max.single.write.to.file.records=1024
 
-# The implementation of the async S3 client to use for upload during ingest.
-# Valid values are 'java' or 'crt'. This determines the implementation of S3AsyncClient that gets used.
-# With 'java' it makes a single PutObject request for each file.
-# With 'crt' it uses the AWS Common Runtime (CRT) to make multipart uploads.
-# Note that the CRT option is recommended. Using the Java option may cause failures if any file is >5GB in size, and
-# will lead to the following warning:
-# "The provided S3AsyncClient is not an instance of S3CrtAsyncClient, and thus multipart upload/download feature is not
-# enabled and resumable file upload is not supported. To benefit from maximum throughput, consider using
-# S3AsyncClient.crtBuilder().build() instead."
-# (async partition file writer only)
-sleeper.ingest.async.client.type=crt
-
-# The part size in bytes to use for multipart uploads.
-# (CRT async ingest only) [128MB]
-sleeper.ingest.async.crt.part.size.bytes=134217728
```

Comment:
It looks like these 2 properties were missed when setting the async upload settings explicitly.